lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb

```diff
@@ -1643,8 +1643,10 @@ def setup_chw_loop_with_components(model, chw_loop, chiller_type)
     sizing_plant.setDesignLoopExitTemperature(7.0)
     sizing_plant.setLoopDesignTemperatureDifference(6.0)
 
-    # pump = OpenStudio::Model::PumpConstantSpeed.new(model)
-    chw_pump = OpenStudio::Model::PumpConstantSpeed.new(model)
+    # Note: pump of 'chilled water loop' has been changed to the variable one as the constant one caused fatal errors for LargeOffice-Yellowknife-NaturalGas for some ECMs and inputs.
+    # Fatal error was: 'CheckForRunawayPlantTemps: Simulation terminated because of run away plant temperatures, too cold' OR '..., too hot' for the PlantLoop of 'Chilled Water Loop'.
```

Comment:
This indicative of a control issue, and just switching to pump variable speed may mask the problem.

Reply:
I also tried to solve this issue through availability manager (AvailabilityManagerLowTemperatureTurnOff and AvailabilityManagerHighTemperatureTurnOff) without changing the pump type, however it did not work. We will investigate it further in the future.

Reply:
ok.  @ckirney @srgilani I'll go ahead and merge but this is something you will want to address.